frontend/src/metabase/dashboard/components/Dashboard/Dashboard.jsx

```diff
@@ -115,8 +115,8 @@ class Dashboard extends Component {
   );
 
   // NOTE: all of these lifecycle methods should be replaced with DashboardData HoC in container
-  componentDidMount() {
-    this.loadDashboard(this.props.dashboardId);
+  async componentDidMount() {
```

Comment:
This has nothing to do with your change, but just a note for the future: we've prefixed all lifecycle methods with `UNSAFE_` at one point. Not sure how did these two slip through.